### ISSUE TITLE
Broken link on hashtag admin detailed view

### ISSUE THREAD

**Issue by Kjwon15** [tootsuite/mastodon#11957]
### Expected behaviour

On `/admin/tags/:id` page that part I circled with red line is linked to `/explore/:tag`
It should be accessible profile directory with hashtag.

![image](https://user-images.githubusercontent.com/5047683/65649402-609e6480-e041-11e9-9069-9fe0ace41713.png)

### Actual behaviour

Hashtag for profile directory is not implemented. That returns _404 Not Found_

### Steps to reproduce the problem

Just go to `/admin/tags/1` and click that link.
It will send you to `/explore/somehashtag` and you'll see  _404 not found_

### Specifications

On mastodon v3.0.0rc1

**Comment by mayaeh** [comment]
@Kjwon15 Does the same error occur if the number in the red circle is not 0?
If someone uses that hashtag in their profile, I don't think that error will occur.

If no one uses that hashtag on their profile, should we return a message to that effect?

**Comment by Kjwon15** [comment]
@mayaeh Yes, profile directory with hashtags are removed and not implemented during rework.
